tests/settings.py

```diff
@@ -0,0 +1,38 @@
+import dj_database_url
+
+SECRET_KEY = "ToP SeCrEt"
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "fcm_django",
+]
+MIDDLEWARE = [
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+]
+
+DATABASES = {"default": dj_database_url.config(default="sqlite://")}
+USE_TZ = True
+ROOT_URLCONF = "tests.urls"
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ],
+        },
+    },
+]
+
+IS_SWAP = False
```

Comment:
Yeah it could be. But in that case we have to introduce developer into how swapper works.

Reply:
If comment comment for this line of code make any sense i could add it.

But if your point that the same thing could be done in different way just because it could be done I don't think that is a constructive.

Reply:
You are making an API for users to consume. Why would you force your users to do additional steps if you can handle this on your side?

Reply:
If it's just for tests, then it does not matter.

Reply:
I added a comment to this line so that there would be no confusion.